components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -267,3 +271,47 @@ function loadPolicyCompliance() {
         }
     );
 }
+
+function loadApplicationsList() {
+    var applicationsList = $("#applications-list");
+    var applicationListingTemplate = applicationsList.attr("src");
+    var deviceId = applicationsList.data("device-id");
+    var deviceType = applicationsList.data("device-type");
+
+    $.template("application-list", applicationListingTemplate, function (template) {
+        var serviceURL = "/api/device-mgt/v1.0/devices/" + deviceType + "/" + deviceId + "/applications";
+        invokerUtil.get(
+            serviceURL,
+            // success-callback
+            function (data, textStatus, jqXHR) {
+                if (jqXHR.status == 200 && data) {
+                    data = JSON.parse(data);
+                    $("#apps-spinner").addClass("hidden");
+                    if (data.length > 0) {
+                        for (var i = 0; i < data.length; i++) {
+                            data[i]["name"] = decodeURIComponent(data[i]["name"]);
+                            data[i]["platform"] = deviceType;
+                        }
+
```

Comment:
Remove the unnecessary blank line.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/950